ta/sims/ta_sims.c

```diff
@@ -135,3 +135,23 @@ TEE_Result sims_get_counter(void *session_context, uint32_t param_types,
 
 	return TEE_SUCCESS;
 }
+
+TEE_Result sims_entry_panic(uint32_t param_types, TEE_Param params[4])
+{
+	volatile bool mytrue = true;
+	(void)param_types;
```

Comment:
minor: add an empty line between variables definition and instructions.

Reply:
Done